cnf-certification-test/platform/suite.go

```diff
@@ -69,23 +69,18 @@ var _ = ginkgo.Describe(common.PlatformAlterationTestKey, func() {
 })
 
 // testContainersFsDiff test that all CUT didn't install new packages are starting
-func testContainersFsDiff(env *provider.TestEnvironment) {
+func testContainersFsDiff(env *provider.TestEnvironment, testerFuncs cnffsdiff.FsDiffFuncs) {
```

Comment:
Just for note, we decided we didn't want to unit test this func because it's simple enough to read and we wanted to keep the direct calls to gomega.